src/components/generic/Modal/Modal.js

```
@@ -137,18 +137,21 @@
   footerContent,
   toolbarContent = undefined,
   maxWidth,
+  padding,
```

ESLint

[error] 'padding' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)